test/Field/Raw/golden/operator-precedence-00.field

```diff
@@ -0,0 +1,12 @@
+	  a * b
+	- c * d
+    ==
+	  e / f
+	+ g / h
+xor
+	  a * b
+	- c * d
+    >=
+	  e / f
+	+ g / h
```

Comment:
It would be really nice to have another file alongside these two where we pretty-print the result of parsing to see how things get formatted. But that can be a separate PR.

Reply:
I will add a pretty-printing golden testing pass as part of #30.